kanidmd/src/lib/value.rs

```diff
@@ -1349,7 +1394,8 @@ impl Value {
             PartialValue::Utf8(s)
             | PartialValue::Iutf8(s)
             | PartialValue::Iname(s)
-            | PartialValue::Nsuniqueid(s) => vec![s.clone()],
+            | PartialValue::Nsuniqueid(s)
+            | PartialValue::EmailAddress(s) => vec![s.clone()],
             PartialValue::Refer(u) | PartialValue::Uuid(u) => {
```

Comment:
Did this pass `cargo fmt`?

Reply:
I'm on a different version of rust 1.47 so it would have been formatted, but may have a different fmt version.